src/skottie-player.ts

```diff
@@ -2,13 +2,17 @@
  * skottie-player.ts
  */
 
-import {LitElement, html, customElement, property, css} from 'lit-element';
-const LottieModule = (window as any).lottie;
-import * as lottie from 'lottie-web/build/player/lottie_light';
-type LottiePlayerRoot = typeof lottie.default;
-const lottiePlayer = (LottieModule as any) as LottiePlayerRoot;
-console.log(lottie);
-console.log(lottiePlayer);
+import {css, customElement, html, LitElement, property} from 'lit-element';
+import lottiePlayer, {AnimationConfigWithPath} from 'lottie-web/build/player/lottie_light';
```

Comment:
Do we still need the Lottie player at this point?

Reply:
No, thanks for catching that.